### ISSUE TITLE
if md_in_html is enabled, render tile captions using markdown

### ISSUE THREAD

**Issue by ntno** [ntno/mkdocs-terminal#57]
### Describe the problem you'd like to have solved
I would like to be able to enter markdown for a tile caption and have it be rendered in the final tile grid

### Describe the ideal solution
if the tile.caption is markdown then it should render correctly when md_in_html extension is enabled

ex: 
tile.caption: `Figure 8, *Misc. Project Notes*, pages 82, 83.`
then 'Misc. Project Notes' should be in italics

## Alternatives and current work-arounds

current workaround is to use html tags
ex:
tile.caption: `Figure 8, <i>Misc. Project Notes</i>, pages 82, 83.`

### Additional context

![Screen Shot 2023-01-09 at 5 59 32 PM](https://user-images.githubusercontent.com/20992605/211425989-67223c6e-4a82-4cfe-9efc-8119e672489d.png)
![Screen Shot 2023-01-09 at 5 59 16 PM](https://user-images.githubusercontent.com/20992605/211425990-05952375-4e64-44fa-b355-fe2db3b7dbab.png)

**Comment by ntno** [comment]
see https://github.com/ntno/ntno.net/blob/use-mkdocs-terminal/docs/electronics/led-backpack.md?plain=1#L4
and http://ntno-net-ci-pr-107.s3-website.us-east-2.amazonaws.com/electronics/led-backpack/#process-photos

**Comment by ntno** [comment]
see https://github.com/WSOFT-Project/lantana/blob/2c2da72d109df641280306b88a3798bbc492c5da/setup.py#L24-L26
maybe need to make an extension?  or add a extension definition to the theme and point to jinja2_markdown

see also: 
- https://gist.github.com/glombard/7554134
- https://stackoverflow.com/questions/23320668/how-to-render-markdown-content-with-jinja2-in-a-django-project
- https://gist.github.com/tamsky/c04d422351a332d2b4e8/revisions

**Comment by ntno** [comment]
- markdown extension has testing framework: https://python-markdown.github.io/test_tools/

**Comment by ntno** [comment]
https://github.com/mkdocs/mkdocs/blob/master/docs/dev-guide/plugins.md

**Comment by ntno** [comment]
https://github.com/byrnereese/mkdocs-markdown-filter

**Comment by ntno** [comment]
https://github.com/timvink/mkdocs-enumerate-headings-plugin

**Comment by ntno** [comment]
#87 
#85

**Comment by ntno** [comment]
# resolved with version 3.7.0
- updated requirements to newest version
- added "md-to-html" in plugins list

## tile config
```yaml
---
grid_css: ntno_img_card_grid
tiles: 
  - caption: 'Figure 8, *Misc. Project Notes*, pages 82, 83.'
    img_src: /img/led-biking-backpack/led-layout-brainstorm.png
    img_title: '' #TODO
    img_alt: '' #TODO
---
```

## mkdocs.yml
```yaml
markdown_extensions:
  - attr_list
  - meta
  - md_in_html
  - toc:
      permalink: "#"
  
plugins:
  - md-to-html
  - search
  - git-revision-date
```

## requirements.txt
```text
mkdocs
mkdocs-terminal>=3.7

# Python Markdown Extensions
pygments>=2.14
pymdown-extensions>=9.9

# MkDocs Plugins
mkdocs-git-revision-date-plugin
mkdocs-exclude-search
```

## result

![Screen Shot 2023-01-21 at 7 51 25 PM](https://user-images.githubusercontent.com/20992605/213895665-65607d42-10cc-4c1c-a40a-cdc749cf02b9.png)

**Comment by ntno** [comment]
implemented new plugin which comes pre-installed with the theme.  enable by adding `md-to-html` or `terminal/md-to-html` in plugins list.  if plugin is not available (ie you don't enable the plugin), markup filter will be skipped during tile rendering.
